packages/keiko-server/src/grounded-faithfulness.ts

```
@@ -538,7 +551,7 @@ export function splitClaimSpans(text: string): readonly string[] {
 /** Remove inline `[...]` citation brackets from a claim span so the judge sees the prose claim. */
 export function stripInlineCitations(text: string): string {
   return text
-    .replace(/\[[^\]\n]{1,200}\]/g, " ")
+    .replace(/[\[［【][^\]］】\n]{1,200}[\]］】]/g, " ")
```

ESLint

[error] Unnecessary escape character: \[.

(no-useless-escape)